src/components/legacy/markdown/custom-components.tsx

```diff
@@ -214,6 +214,13 @@ const customSatellytesComponents = {
     );
   },
   p(props) {
+    const hasVideo = React.Children.toArray(props.children).some(
+      (child) =>
+        React.isValidElement(child) && typeof child.props?.videoId === 'string',
+    );
+    if (hasVideo) {
+      return <>{props.children}</>;
+    }
```

Comment:
Nice solution to this problem! I added one suggestion to simplify the code a little bit. I also noticed that the bottom margin is missing. Can you add that? Maybe the same as for the images in a blog post

<img width="1229" alt="Screenshot 2024-11-20 at 14 19 31" src="https://github.com/user-attachments/assets/6490324b-4987-4cbf-b9fd-ac10635f9c00">


```suggestion
    const hasVideo = React.Children.toArray(props.children).some(
      (child) => React.isValidElement(child) && child.props?.videoId,
    );
    if (hasVideo) {
      return <>{props.children}</>;
    }
```